test/fixtures/detect-existing-browser-api-object/copy-original-api-objects.js

```diff
@@ -0,0 +1,2 @@
+// Store a copy of the references to the original API objects.
+const originalAPIObjects = {browser, chrome}; // eslint-disable-line no-unused-vars
```

Comment:
Add a comment that in browsers without a `browser` API, this will point to the `<div id="browser">` in the page.